Automattic/wp-calypso
# Color Schemes: Use --sidebar variables for colors on mobile devices

#### Changes proposed in this Pull Request

* On mobile devices, the menu colors were using direct color variables; this updates the menu items to use `--sidebar-background` and the like, to be compatible with color schemes.
* This does change some colors on mobile devices, which is why I'm requesting a design review. 

**Before**

<img width="399" alt="screen shot 2019-02-12 at 1 38 52 pm" src="https://user-images.githubusercontent.com/2124984/52660399-94bb7f80-2ecd-11e9-9e80-e873cf7b8fca.png">

**After**

<img width="400" alt="screen shot 2019-02-12 at 1 39 28 pm" src="https://user-images.githubusercontent.com/2124984/52660405-99803380-2ecd-11e9-869c-288a5474ac32.png">

#### Testing instructions

* Switch to this PR and ensure you're working on a mobile device or emulator
* Check out the sidebar menu in My Sites, Reader, Account Settings, etc. to see the changes.
* Check for contrast issues or visual bugs in all available color schemes.
